.travis.yml

```diff
@@ -5,9 +5,15 @@ rvm:
 
 before_install:
 - git clone https://github.com/puppetlabs/puppetlabs-ruby_task_helper ../ruby_task_helper
+- curl -fSL "https://releases.hashicorp.com/terraform/0.12.13/terraform_0.12.13_linux_amd64.zip" -o terraform.zip
```

Comment:
I wonder if I can snap install. I wish there was a `terraform latest` url...

Reply:
We could, though then we'd have to make sure that the snap package stays up to date :/ https://snapcraft.io/terraform hasn't been updated since last January